ngx-tools/src/define-type/define-type.ts

```diff
@@ -7,35 +7,38 @@ let typeCache: {[label: string]: boolean} = {};
 /**
  * Ensure you only define an action once in the entirety of the application
  *
- * @param label The action label
- * @return uniqueLabel The unique label
+ * @param { String } label: The action label
```

Comment:
We don't need the types defined in our comments since they are defined in the function params directly.

Also, we've been separating our param name and description with a dash rather than colon:

```
* @param label - The action label
```